src/core/less/base.less

```diff
@@ -65,10 +65,13 @@
         right:inherit;
         left:-320px;
     }
+    z-index:501;
+    .drawer-holder {
+        height: 100%;
```

Comment:
When moving height into `drawer-holder` we lose the 100% height on the drawer.

Reply:
Not sure that's correct. `drawer` maintains a rule for height: 100% at line 61. And shows up at run time in my dev tools. Am I misunderstanding?